recipes/date/all/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.11)
+cmake_minimum_required(VERSION 3.7)
```

Comment:
Please leave that at `2.8`

There is no reason to raise the version for this CMake file.

If `data` requires a newer version CMake will recognize this still

Reply:
Upstream CMakeLists.txt requires 3.7. Will it work if our CMakeLists.txt is on 2.8?

Reply:
If so, is there a policy on specifying a version in CMakeLists.txt of recipe and test_package? Should it be 2.8.11, 2.8.12 or 3.1?

Reply:
> Upstream CMakeLists.txt requires 3.7. Will it work if our CMakeLists.txt is on 2.8?

Yes. As I tried to say CMake is collecting all `cmake_minimum_required` versions at runtime and compares them individual to the currently executing CMake version, they do not overwrite each other

> If so, is there a policy on specifying a version in CMakeLists.txt of recipe and test_package? Should it be 2.8.11, 2.8.12 or 3.1?

Implicitly through our hooks. Maybe we should write a summary for this somewhere.

recipe CMakeLists.txt: `2.8.11` or `2.8.12` (doesn't really matter much either way)  - except when you have to use `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` then it has to be `3.4`

test_package CMakeLists.txt: `3.1`

Reply:
Thanks for your explanations! Restored main version